CHANGELOG.md

```diff
@@ -0,0 +1,97 @@
+# Changelog
+
+[`tinytga`](https://crates.io/crates/tinytga) is a no_std, low memory footprint TGA loading library for embedded applications.
+
+<!-- next-header -->
+
+## [Unreleased] - ReleaseDate
+
+### Changed
+
+- **(breaking)** [#407](https://github.com/embedded-graphics/embedded-graphics/pull/407) The `image_descriptor` in `TgaHeader` was replaced by `image_origin` and `alpha_channel_bits`.
```

Comment:
Should we add a note about the changed PR numbering for future changes, because the repo has changed?

Reply:
Yes - see other changelog comment and addition.